bundles/af-core/src/test/resources/form/formcontainer/datasource/test-content.json

```diff
@@ -20,5 +20,14 @@
                 "jcr:primaryType": "nt:unstructured"
             }
         }
+    },
+    "formattertypedatasource": {
+        "jcr:primaryType"   : "nt:unstructured",
+        "datasource" : {
+            "sling:resourceType": "core/fd/components/form/container/v1/datasource",
+            "type": "formatters",
+            "guideDataModel": "basic",
```

Comment:
do we need guideDataModel? we shouldn;t ideally unless there's a bug

Reply:
removed the,